content/hardware/03.nano/boards/nano-esp32/tutorials/cheat-sheet/cheat-sheet.md

```diff
@@ -430,7 +432,7 @@ To send and receive data through UART, we will first need to set the baud rate i
 
 ### UART
 
-The pins used for UART on the Nano ESP32 are the following:
+The default pins for UART communication on the Nano ESP32 are the following:
```

Comment:
I would like to stress (here or somewhere else - maybe even changing the UART title above) that this `Serial0` UART is shared with the bootloader/kernel and thus does have some I/O that is **not controllable by the sketch** (not even by me... :man_shrugging:):
* at each boot/reset, a few messages are printed by the bootloader;
* if something goes horribly wrong, a crash dump is printed by FreeRTOS.

We should encourage everyone to use `Serial1`/`Serial2` to avoid this issue - several have complained in the forums.